lms/static/sass/discussion/_discussion.scss

```diff
@@ -808,35 +499,13 @@ body.discussion {
               color: #888;
             }
 
+            // deprecated styles - remove after review / cleanup
             header {
-              padding-bottom: 0;
-              margin-bottom: ($baseline*0.75);
-
-              .posted-details {
-                margin-top: ($baseline/5);
-
-                .username {
-                  display: inline;
-                  font-size: 14px;
-                  font-weight: 700;
-                }
-              }
-
-              h1 {
-                font-size: 19px;
-                font-weight: 700;
-                margin-bottom: 0px !important; // Override courseware CSS
-              }
 
               h4 {
```

Comment:
Author To-Do: I couldn't find where this h4 is used, which is why this whole section isn't gone yet. I will look again to see where this might be used prior to merging.